hack/verify-jsonformat.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+json_files=$(find {api,examples,docs,images,plugins,test} -name "*.json")
+tmp_dir=$(mktemp -d --suffix=jsonformat)
+found=0
+
+set +e
+
+enable_formatting="${1:-""}"
+fix_formatting="${2:-""}"
+
+for f in $json_files; do
+  tmp_file="${tmp_dir}$(basename $f)"
+  go run ./hack/jsonformat.go ${f} > ${tmp_file}
+  result=$?
+  if [ "${enable_formatting}" == "--with-formatting" ]; then
+    if ! diff --brief ${f} ${tmp_file} > /dev/null; then
+      if [ "$result" == "0" -a "${fix_formatting}" == "--fix" ]; then
+        echo "Fixing JSON formatting: '${f}'"
+        cp ${tmp_file} ${f}
+      else
+        echo "Invalid JSON formatting: '${f}'"
+        found=1
+      fi
+      continue
+    fi
+  fi
+  [ "$result" != "0" ] && found=1
+done
+
+rm -rf ${tmp_dir}
+
+if [ "$found" == "1" ]; then
+  echo -e "\nRun: 'go run ./hack/jsonformat.go <filename>'\n"
```

Comment:
This is because the file contains errors, right?  this message could be more informational :)